dmn-io-xml/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>dmn</artifactId>
+        <groupId>org.powerflows</groupId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>dmn-io-xml</artifactId>
+
+    <properties>
+        <jaxb.version>2.3.1</jaxb.version>
+    </properties>
```

Comment:
Arent we moving props to parent pom?

Reply:
We are, but this was before our discussion... I'll change

Reply:
Done